GVFS/GVFS/CommandLine/CloneVerb.cs

```diff
@@ -121,7 +121,9 @@ public override void Execute()
                 CacheServerInfo cacheServer = null;
                 ServerGVFSConfig serverGVFSConfig = null;
 
-                using (JsonTracer tracer = new JsonTracer(GVFSConstants.GVFSEtwProviderName, "GVFSClone"))
+                string mountId = CreateId();
+                string enlistmentId = CreateId();

```

Comment:
@kewillford I had another look at setting enlistmentId/mountId before the clone.  

Mountid is pretty easy to set as we can just pass it in to LogEnlistmentInfoAndSetConfigValues into this class.  

EnlistmentId however is more interesting.   We want to grab Id before the enlistment actually exists.  You can see from the associated commit how I did it but it required some tweaking.  

So I guess my summary is:
  - mountId easy to pass thru, but would look weird in logs without enlistmentId
  - enlistmentId possible to pass thru, but required some class tweaking